portfolio/src/main/webapp/script.js

```diff
@@ -59,9 +59,50 @@ function createListElement(task) {
   return liElement;
 }
 
+var map;
```

Comment:
This doesn't need to be a global variable, right?

Reply:
Jenny told me to so that it's possible to reference in any scope level